architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,23 @@ endif::openshift-dedicated,openshift-online[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+[[architecture-projects-openshift-default-project]]
+===The `openshift` and `default` projects
```

Comment:
Describing the "openshift" namespace has value.  It is special, user-facing, and logically cluster-admin maintained.  It now interacts with the templatservicebroker @jim-minter too.